CHANGELOG.md

```diff
@@ -57,6 +58,7 @@ Please visit [cucumber/CONTRIBUTING.md](https://github.com/cucumber/cucumber/blo
 ### Fixed
 
 *  Fix the handling of failed hooks in the JUnit Formatter ([@brasmusson](https://github.com/brasmusson))
+>>>>>>> master
```

Comment:
TODO: This is a conflict marker, which should not be in the resulting file.

Reply:
All set on this. Thanks for the eye!